etc/kayobe/ansible/deploy-os-capacity-exporter.yml

```diff
@@ -1,27 +1,51 @@
 ---
-- hosts: monitoring
-  gather_facts: false
 
+- name: Remove legacy os_exporter.cfg file
+  hosts: network
+  gather_facts: false
   tasks:
     - name: Ensure legacy os_exporter.cfg config file is deleted
       ansible.builtin.file:
         path: /etc/kolla/haproxy/services.d/os_exporter.cfg
         state: absent
-      delegate_to: network
       become: true
+      when: stackhpc_enable_os_capacity
```

Comment:
It could be helpful to remove this file unconditionally, given that when OS capacity is disabled it writes out an empty file.

Reply:
Removed conditonal